README.md

```diff
@@ -17,6 +17,7 @@ The following topics will be covered in this readme
 * Show Campaign / Send Event
 * Callback from SDK
```

Comment:
If we want to make these "topics" linked to their section in the same document we should use the markdown construct [name of the topic](#name of the topic below in the document)